pyam/core.py

```diff
@@ -1531,7 +1531,16 @@ def bar_plot(self, *args, **kwargs):
         df = self.as_pandas(with_metadata=True)
         ax = plotting.bar_plot(df, *args, **kwargs)
         return ax
+        
+    def boxplot(self, *args, **kwargs):
```

Comment:
can this be `box_plot` to match other functions please?

Reply:
I disagree, I would follow the pandas convention where it's also called `boxplot`